Symfony/CS/Tests/Fixer/PSR2/FunctionCallSpaceFixerTest.php

```diff
@@ -80,6 +80,15 @@ public function testFixProvider()
             array(
                 '<?php $a = 2 * (1 + 1);',
             ),
+            array(
+                '<?php echo ($a == $b) ? "foo" : "bar";',
+            ),
+            array(
+                '<?php echo ($a == test($b)) ? "foo" : "bar";',
+            ),
+            array(
+                '<?php include ($html)? "custom.html": "custom.php";',
```

Comment:
=) Failures. Possible we can check if we have `echo` `include` `include_once` `require` `require_once` and `print` then check if the next token is `?` 
It will fix this bug and leave current reformat logic. 
Your Thought?